Library/Invoice.cs

```diff
@@ -104,6 +105,16 @@ public byte[] GetPdf(string acceptLanguage = "en-US")
         }
 
         /// <summary>
+        /// Returns a PDF representation of an invoice
+        /// </summary>
+        /// <param name="acceptLanguage">Language for invoice, defaults to en-US.</param>
+        /// <returns></returns>
+        public async Task<byte[]> GetPdfAsync(string acceptLanguage = "en-US")
```

Comment:
Yeah, hate to be a bother but I'd prefer for it to just have the commits intended. Keeps the history clean.